src/components/layout/Footer.tsx

```diff
@@ -2,11 +2,10 @@ const Footer = () => {
     return (
       <footer
       role="contentinfo"
-      className="bg-background-footer text-text-primary py-8 w-full"
-    >
-        <p className="text-center">&copy; 2024 Font Hill Records. All rights reserved.</p>  
-      
-      </footer>
+      className="bg-background-footer text-text-primary py-8 w-full">
+        <p className="text-center">© 2024 Font Hill Records. All rights reserved.</p>
```

Comment:
```suggestion
        <p className="text-center">&copy; 2024 Font Hill Records. All rights reserved.</p>
```
We should keep using normal html notation for symbols, why the change?